modules/showtime.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-return-assign */
```

Comment:
- [ ] Kindly don't disable the linters since they are very important to improve your code and make it DRY KISS and clean since it is about some rules to organize your code so kindly don't disable them and try to solve the issues instead of disabling them✨🚀💯😊🔥